spec.md

```diff
@@ -111,7 +111,7 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| services* | array-of-service-objects | Schema of service objects defined below. |
+| services* | array-of-service-objects | Schema of service objects defined below. MUST contain at least one service. |
 
```

Comment:
I think as the type is a list of services, specifying at least one service makes sense.